src/common/meta/src/datanode.rs

```diff
@@ -92,8 +92,22 @@ pub struct RegionStat {
     pub sst_size: u64,
     /// The size of the SST index files in bytes.
     pub index_size: u64,
-    /// The version of manifest.
-    pub manifest_version: u64,
+    /// The details of the region.
+    pub detail: RegionDetail,
```

Comment:
`RegionManifest` ?

Reply:
resolved in [4e305ef](https://github.com/GreptimeTeam/greptimedb/pull/5811/commits/4e305ef13e856a6ceb2ad7591737013284bc0fb3)